### DISCUSSION TITLE
Cant get voiceattack to change frequency for com 1

### DISCUSSION THREAD

**Discussion by janeike** [jamescl604/MSFSVoiceAttackPlugin#12] [Q&A]
hello. Donno if I have misunderstood anything regarding this. I have tried several ways of speaking to VA, like:

"Set com 1 to 200"
It wont work. (as soon as I said "to", It reports "not set".)
Attaced some pics.

Any Ideas howto get this to work?

Regards Jan
![radios dashboard](https://user-images.githubusercontent.com/87079947/125163704-aaacf780-e18e-11eb-8a01-b218b8f78eda.png)
![VA](https://user-images.githubusercontent.com/87079947/125163705-ab458e00-e18e-11eb-8117-05557d3fa8f5.png)
![VAcommand1 expanded](https://user-images.githubusercontent.com/87079947/125163706-ab458e00-e18e-11eb-9c66-99e1874a92b6.png)
![VAcommand1](https://user-images.githubusercontent.com/87079947/125163707-abde2480-e18e-11eb-9936-139983aba200.png)

**Comment by Raging-Beard** [comment]
Did you ever get this working? Thanks.

**Reply by jimmyap** [reply]
I've yet to see someone that can set radios for this.
